src/cast/const.ts

```diff
@@ -1,11 +1,11 @@
 // Guard dev mode with `__dev__` so it can only ever be enabled in dev mode.
 export const CAST_DEV = __DEV__ && true;
 // Replace this with your own unpublished cast app that points at your local dev
-const CAST_DEV_APP_ID = "5FE44367";
+const CAST_DEV_APP_ID = "5280DF74";
```

Comment:
lol you prefer your local dev instead of mine? :P

Reply:
Can we extract the dev constants into their own file so we can keep the things like `CAST_NS` and `CAST_APP_ID` in a non-ignored file.